docs/topics/Gateway.md

```diff
@@ -786,6 +801,49 @@ Sent when a channel is updated. The inner payload is a [channel](#DOCS_RESOURCES
 
 Sent when a channel relevant to the current user is deleted. The inner payload is a [channel](#DOCS_RESOURCES_CHANNEL/channel-object) object.
 
+#### Thread Create
+
+Sent when a thread is created, relevant to the current user, or when the current user is added to a thread. The inner payload is a [channel](#DOCS_RESOURCES_CHANNEL/channel-object) object.  When being added to an existing private thread, includes a [thread member](#DOCS_RESOURCES_CHANNEL/thread-member-object) object.
+
+#### Thread Update
+
+Sent when a thread is updated. The inner payload is a [channel](#DOCS_RESOURCES_CHANNEL/channel-object) object. This is not sent when the field `last_message_id` is altered. To keep track of the last_message_id changes, you must listen for [Message Create](#DOCS_TOPICS_GATEWAY/message-create) events.
+
+#### Thread Delete
+
+Sent when a thread relevant to the current user is deleted. The inner payload is a subset of the [channel](#DOCS_RESOURCES_CHANNEL/channel-object) object, containing just the `id`, `guild_id`, `parent_id`, and `type` fields.
+
+#### Thread List Sync
+
+Sent when the current user _gains_ access to a channel.
```

Comment:
Does this have no payload? Perhaps saying which channel was joined would be more useful?

Reply:
It's listed below :)